src/components/Rating/RatingComponent.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ARTICLE from '../../mock/article';
+import MyRatingContainer from '../../containers/Rating/MyRatingContainer';
+import AverageRatingContainer from '../../containers/Rating/AverageRatingContainer';
+
+class RateArticle extends Component {
+  componentWillMount() {
+    const { slug, getRatings } = this.props;
+    getRatings(slug);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
The Fragment wrapper is not necessary since you are already wrapping everything in your container div

Reply:
The container div does not wrap everything